gaia-header.js

```diff
@@ -1,29 +1,46 @@
 ;(function(define){'use strict';define(function(require,exports,module){
+/*jshint laxbreak:true*/
 /*jshint esnext:true*/
+/*jshint boss:true*/
```

Comment:
The only problem with that is that the .jshintrc may not always be distributed with the library.